packages/theme-base/src/themes/sap_belize/base-parameters.less

```diff
@@ -290,6 +290,10 @@
 @sapToolbar_SeparatorColor: fade(@sapPrimary1, 20);
 @sapHighlightTextColor: contrast(@sapHighlightColor, @sapTextColor, @sapContent_ContrastTextColor, @sapContent_ContrastTextThreshold);
 
-// custom added params as no base ("sap" prefix) params equivalent exist
 @sapToggleButton_Pressed_HoverBackground: lighten(@sapButton_Selected_Background, 10);
-@sapToggleButton_Pressed_HoverBorderColor: @sapToggleButton_Pressed_HoverBackground;
\ No newline at end of file
+@sapToggleButton_Pressed_HoverBorderColor: @sapToggleButton_Pressed_HoverBackground;
+
+@sapTimePicker_ItemBorderColor: lighten(@sapList_BorderColor, 10);
+@sapTimePicker_ItemColor: lighten(@sapContent_ForegroundColor, 5);
+@sapTimePicker_ItemHoverColor: darken(@sapContent_ForegroundColor, 5);
```

Comment:
Look at the visual design once again,  in the section "Slider Item D1" the only color calculations are for the background
of the expanded item:

fiori_3
fiori_3_dark: darken(@sapUiListBackground, 2);
hcb: background: lighten(@sapUiListBackground, 18);
hcw: background: darken(@sapUiListBackground, 15);

And, this makes one parameter @sapTimePicker_Item_Background_Color


For the Collapsed Slider Item there is one more calculation for the hover state:
background: lighten(@sapUiListActiveBackground, 3);, skip it and set it to sapUiListActiveBackground.

Note:
Instead of sapUiListHoverBackground,  as we removed the global params, 
you have to use sapList_Hover_Background base param.

Instead of sapUiListBackground as we removed the global params, 
you have to use  sapList_Background base param.

Instead of sapUiListActiveBackground, you have to use sapList_Active_Background

and so on, all the params are in place in the base-paramerters.css but keep in mind that their names includes "_" at places and the "Ui" prefix is removed.